app/src/main/java/com/jnj/vaccinetracker/visit/zscore/InputFilterMinMax.kt

```diff
@@ -0,0 +1,30 @@
+package com.jnj.vaccinetracker.visit.zscore
+
+import android.text.InputFilter
+import android.text.Spanned
+
+class InputFilterMinMax(private val min: Int, private val max: Int) : InputFilter {
+
+    override fun filter(
+            source: CharSequence?,
+            start: Int,
+            end: Int,
+            dest: Spanned?,
+            dstart: Int,
+            dend: Int
+    ): CharSequence? {
+        try {
+            val input = (dest.toString() + source.toString()).toInt()
+            if (isInRange(min, max, input)) {
+                return null
+            }
+        } catch (nfe: NumberFormatException) {
+            // Handle the exception
```

Comment:
Are you planning to log an exception message, or adding something like `Log.e("InputFilterMinMax", NumberFormatException: ${nfe.message}")`

Reply:
good point